libs/browser-settings/src/lib/BrowserSettingsProvider.tsx

```diff
@@ -0,0 +1,47 @@
+import { createReducerContext } from 'react-use'
+import { composeRight } from 'ts-pipe-compose'
+
+interface BrowserSettingsState {
```

Comment:
This was moved to a separate lib out of the account provider so that it could be used elsewhere in the context tree (and, to keep things simple in account-land)